packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -11,36 +11,36 @@ const keyCodes = {
     space: 32,
 };
 
-describe("MultiSelectMenu", () => {
-    const onClick = jest.fn();
-    const onToggle = jest.fn();
-    const onChange = jest.fn();
-    const menu = mount(
-        <ActionMenu
-            items={[
-                {
-                    type: "action",
-                    label: "Action",
-                    onClick: () => onClick(),
-                },
-                {
-                    type: "separator",
-                },
-                {
-                    type: "select",
-                    label: "Toggle",
-                    onClick: () => onToggle(),
-                    value: "toggle",
-                },
-            ]}
-            menuText="Action menu!"
-            onChange={(selectedValues) => onChange()}
-            selectedValues={[]}
-        />,
-    );
-    const opener = menu.find("ActionMenuOpener");
-
+describe("ActionMenu", () => {
     it("closes/opens the menu on mouse click, space, and enter", () => {
+        const onClick = jest.fn();
+        const onToggle = jest.fn();
+        const onChange = jest.fn();
+        const menu = mount(
+            <ActionMenu
+                items={[
+                    {
+                        type: "action",
+                        label: "Action",
+                        onClick: () => onClick(),
+                    },
+                    {
+                        type: "separator",
+                    },
+                    {
+                        type: "select",
+                        label: "Toggle",
+                        onClick: () => onToggle(),
+                        value: "toggle",
+                    },
+                ]}
+                menuText={"Action menu!"}
+                onChange={(selectedValues) => onChange()}
+                selectedValues={[]}
+            />,
+        );
+        const opener = menu.find("ActionMenuOpener");
```

Comment:
Good call moving this into this `it`.  I didn't realize that you had this code in the `describe`.  You should almost never have code inside a `describe` that isn't in an `it`.